packages/browserslist-config/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/browserslist-config",
+	"version": "2.0.0",
+	"description": "WordPress Browserslist Shared Config",
+	"author": "WordPress",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"browserslist",
+		"browserslist-config"
+	],
+	"homepage": "https://github.com/WordPress/packages/tree/master/packages/browserslist-config/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/packages.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/packages/issues"
+	},
+	"main": "index.js",
+	"engines": {
+		"node": ">=8"
+	},
+	"devDependencies": {
+		"browserslist": "^2.11.3"
+	},
+	"peerDependencies": {
```

Comment:
Is it necessary since it is already listed as `devDependencies`? Or `devDependencies` should be `dependencies`? I never know :)

Reply:
As `browserslist` is only used in the Jest tests it only requires to be included in `devDependencies`, that said, here's why it is included:

I added it to `peerDependencies` in https://github.com/ntwb/browserslist-config-wordpress/commit/8c6f1b10d9a0c0d43fb2ec95836ac9ae5ac98132 because of `eslint-plugin-node`'s [`no-unpublished-require`](https://github.com/mysticatea/eslint-plugin-node/blob/b0618d4a21a5ce5303f58e125f28705d4d3f7b47/docs/rules/no-unpublished-require.md) rule. 

I opened an issue a while ago [here](https://github.com/mysticatea/eslint-plugin-node/issues/47) asking that `devDependencies` modules be ignored from these rules and the end result was not what I had hoped for, hence the addition of `browserslist` to `peerDependencies` works as a workaround.


These are the two ESLint errors thrown when `browserslist` is not included in `peerDependencies`
```
  1:31  error  "browserslist" is not found                       node/no-missing-require
  1:31  error  "browserslist" is not published                   node/no-unpublished-require
```

I'm going to remove it from `peerDependencies` and add an `.eslintrc.json` in the `test` folder to ignore those two `eslint-plugin-node/recommended` rules as suggested [here](https://github.com/mysticatea/eslint-plugin-node/issues/47#issuecomment-335606914) which I think is not quite as hacky...

Reply:
I removed it https://github.com/WordPress/packages/pull/69/commits/0dfaed0b902ff6cc8a89e0ca78fcf06b272b7f87, will add the `. eslintrc.json` when adding ESLint  in, I'm tempted to turn both those rules off by default config now.